operator/controllers/execution/scheduledscan_controller_test.go

```diff
@@ -1,6 +1,10 @@
 // SPDX-FileCopyrightText: the secureCodeBox authors
 //
 // SPDX-License-Identifier: Apache-2.0
+
+//go:build fast
+// +build fast
```

Comment:
What does this do?

Reply:
Unit tests are implemented that use polling to test if the cronjob has started. This is slower by a maximum of 60 seconds and a minimum of 0 seconds. the slow test has the golang build tag "slow" and other/fast tests have the tag "fast".
Calling make test runs all tests i.e "slow and fast"
Calling make test-fast runs only the tests with "fast" tag.

Reply:
Ah cool didn't know about that 🙌